src/Caching/StackExchangeRedis/src/StackExchangeRedisCacheServiceCollectionExtensions.cs

```diff
@@ -32,4 +33,27 @@ public static IServiceCollection AddStackExchangeRedisCache(this IServiceCollect
 
         return services;
     }
+
+#if NET7_0_OR_GREATER
+    /// <summary>
+    /// Adds Redis distributed caching services to the specified <see cref="IServiceCollection" />.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+    /// <param name="setupAction">An <see cref="Action{RedisCacheOptions}"/> to configure the provided
+    /// <see cref="RedisCacheOptions"/>.</param>
+    /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
+    public static IServiceCollection AddStackExchangeRedisOutputCache(this IServiceCollection services, Action<RedisCacheOptions> setupAction)
+    {
+        ArgumentNullThrowHelper.ThrowIfNull(services);
+        ArgumentNullThrowHelper.ThrowIfNull(setupAction);
+
+        services.AddOptions();
+
+        services.Configure(setupAction);
+        // replace here (Add vs TryAdd) is intentional and part of test conditions
+        services.AddSingleton<AspNetCore.OutputCaching.IOutputCacheStore, RedisOutputCacheStoreImpl>();
```

Comment:
that's because of the dual-TFM build; to do that, we'd need a `#if` around a `using` directive at the top, which IMO is more noise; I will, however, add a comment explaining that